django/thunderstore/repository/migrations/0029_create_namespaces_for_teams.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.1.7 on 2021-11-04 20:32
+
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    Namespace = apps.get_model("repository", "Namespace")
+    Team = apps.get_model("repository", "Team")
+    teams = Team.objects.all().order_by("id")
+    for team in teams:
+        Namespace.objects.create(team=team, name=team.name)
+
+
+def backwards(apps, schema_editor):
```

Comment:
The backwards clause here is both inefficient (not using bulk delete) and unnecessary (the table will be dropped anyway, so no need to individually delete the rows). In other words, just get rid of it.

Reply:
Doned